### PR TITLE
Lmfp heating

### PR DESCRIPTION
**Pull request by uptonsanderbeck**

### CODE REVIEW

**Comment by uptonsanderbeck** [comment]
I don't know if putting rho_crit_baryon in init_cooling in cooling.c is the best place for it, but it seemed the least invasive way to implement it.

File: libgadget/cooling.h [hunk 1]
```diff
@@ -30,6 +30,7 @@ struct cooling_units
 
 /*Initialise the cooling module.*/
 void init_cooling(char * TreeCoolFile, char * MetalCoolFile, char * reion_hist_file, struct cooling_units cu, Cosmology * CP);
+double rho_crit_baryon;
```
Comment (sbird) [inline libgadget/cooling.h]: Could you please make this a member of cooling_units?

Comment (uptonsanderbeck) [reply libgadget/cooling.h]: Done, is this what you had in mind?

**Comment by sbird** [comment]
Yup, looks good! I will merge when travis goes green

**Pull request merged**